templates/todo/projects/csharp-cosmos-sql/.repo/bicep/azure.yaml

```diff
@@ -3,12 +3,37 @@
 name: todo-csharp-cosmos-sql
 metadata:
   template: todo-csharp-cosmos-sql@0.0.1-beta
+workflows:
+  up: 
+    steps:
+      - azd: provision
+      - azd: deploy
```

Comment:
If I understand correctly, this is trying to avoid the default workflow of:

```yaml
- azd: package
- azd: provision
- azd: deploy
```

In which, the last execution of `azd deploy` actually reuses the packaged content in the first step: `azd package`. Just wanted to confirm the intention here, don't have any coding suggestions.

Reply:
Yes,
I had an alternative re-defining `azd up` for all todo-templates which was depending on a new azd feature that was allowing to break the cache from package to deploy (see: https://github.com/Azure/azure-dev/pull/3177). But the feature was not approved.

The recommendation was to use workflows to re-define azd up.